### ISSUE TITLE
Using anissete server results in apple forcing password reset

### ISSUE THREAD

**Issue by Jark5455** [NyaMisty/AltServer-Linux#88]
Whenever I sign my app with the anissete server, the app doesnt get signed but I get a prompt that my icloud account has been locked  for security reasons and the password must be reset.

**Comment by PARKasd** [comment]
any solution? Maybe ansiette data has been blocked by Apple. Whenever I try this with lots of accounts, all the accounts has been locked

**Comment by Jark5455** [comment]
It appears to be an issue with nyamisties specific anisette server,  using sideloadlies anissette server and using [this](https://github.com/Dadoum/Provision) appear to be working fine

**Comment by PARKasd** [comment]
Yes but that anisette server seems to ask 2FA frequently. Using Provision will be safer but pretty annoying

**Comment by alf1e** [comment]
Any updates on this?

**Comment by MintyGitOut** [comment]
11-12 months later and this is _still_ an issue, I opened another issue for this because it's honestly pretty unacceptable

**Comment by bitxeno** [comment]
you can use the custom Anissete server : https://github.com/Dadoum/Provision, solve this issue
